content/homepage.js

```diff
@@ -0,0 +1,40 @@
+import { FaDiscord, FaPlayCircle } from "react-icons/fa";
+
+export default {
+  hero: {
+    eyebrow: "What is 4C?",
+    title: (
+      <>
+        The Cool Community For{" "}
+        <strong className="text-orange-600">Content Creators</strong>
+      </>
+    ),
+    description: (
+      <>
+        <p>
+          Creating content can be rewarding, but it can also be a challenge to
+          come up with ideas, get your content out there, and get it noticed.
+        </p>
+        <p>
+          Join our online community to connect with other like-minded creators,
+          collaborate on ideas, and motivate each other to create and publish!
+        </p>
+      </>
+    ),
+    primaryLink: (
+      <a
+        href="https://discord.com/invite/TcmA2kbJeA"
+        className="btn btn-primary"
+      >
+        <FaDiscord />
+        Join the Community!
+      </a>
+    ),
+    secondaryLink: (
+      <a href="https://discord.com/invite/TcmA2kbJeA" className="btn">
```

Comment:
This also doesn't go anywhere. I've mentioned this before for the same component - pretty sure this is the one being used so there may be a redundant/duplicate somewhere?

EDIT: Yep, Old_join was kept as a reference but that can probably be deleted now - everything was merged into a single component (Hero)